docs/reference/commandline/run.md

```diff
@@ -195,12 +195,13 @@ a container writes files. The `--read-only` flag mounts the container's root
 filesystem as read only prohibiting writes to locations other than the
 specified volumes for the container.
 
-    $ docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v ./static-docker:/usr/bin/docker busybox sh
+    $ docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v /path/to/static-docker-binary:/usr/bin/docker busybox sh
 
 By bind-mounting the docker unix socket and statically linked docker
-binary (such as that provided by [https://get.docker.com](
-https://get.docker.com)), you give the container the full access to create and
-manipulate the host's Docker daemon.
+binary (such as that provided by [Docker install from binary](
+https://docs.docker.com/engine/installation/binaries/#get-the-linux-binary)),
```

Comment:
Oh, sorry, I was lazy copying the link. Given that this is in the same repository, can you make this a relative link, so that it works both on github and in the documentation? Should probably be something like;

`../../installation/binaries.md#get-the-linux-binary` but you may want to test if that's correct (both on GitHub, and by running `make docs`)